test/system/capability.test.js

```diff
@@ -1,16 +1,16 @@
 const test = require('ava')
 const goshawkdb = require('../..')
-const {setupThenTransactionTest, firstRoot} = require('../helpers/utils')
+const {setupThenTransactionTest, testRootName} = require('../helpers/utils')
 
 test("Capabilities are enforced.",
 	setupThenTransactionTest(
 		(t, connection, txn) => {
-			const root = firstRoot(txn)
+			const root = txn.roots[testRootName]
```

Comment:
I.e. I'm guessing you just get back undefined. So then should the test explicitly test for non-undef? I think the tests in the other clients have an explicit helper to get the test root and check it exists.